languages/ruby/maintainers.md

```diff
@@ -8,4 +8,4 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+- Pablo Vicente (Slack: `@pvcarrera`, GitHub: `@pvcarrera`)
```

Comment:
```suggestion
- Pablo Vicente (Slack: `@pvcarrera`, GitHub: `[pvcarrera](https://github.com/pvcarrera)`)
```
Adjusting it to the style in the csharp repo.

Reply:
👍

Reply:
Can you apply the change?

Reply:
No, I don't have permissions

Reply:
I mean this button above: "Commit suggestion"
![image](https://user-images.githubusercontent.com/6624571/72462358-354aab80-37d1-11ea-9b8a-884b9964e4fb.png)

Reply:
I can add another commit with your suggestion but I can't apply your suggestion on GitHub